DQM/L1TMonitorClient/python/L1TStage2BMTFQualityTests_cfi.py

```diff
@@ -0,0 +1,15 @@
+# quality tests for L1T Stage2 BMTF 
+ 
+import FWCore.ParameterSet.Config as cms
+
+l1TStage2BMTFQualityTests = cms.EDAnalyzer("QualityTester",
```

Comment:
Currently the QualityTester class is still an edm::EDAnalyzer and was not changed by PR 18751 , so I am not sure if the class should be changed to a DQMEDHarvester.